src/azure-cli/azure/cli/command_modules/resource/tests/latest/sample_policy_set.json

```diff
@@ -1 +1 @@
-[{"policyDefinitionId": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Authorization/policyDefinitions/azure-cli-test-policy000002", "parameters": {"allowedLocations": {"value": ["australiaeast", "eastus", "japaneast", "westus"]}}}, {"policyDefinitionId": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Authorization/policyDefinitions/azure-cli-test-data-policy000004"}]
\ No newline at end of file
+[{"policyDefinitionId": "/providers/Microsoft.Management/managementgroups/cli-test-mgmt-group000002/providers/Microsoft.Authorization/policyDefinitions/azure-cli-test-policy000003", "parameters": {"allowedLocations": {"value": ["australiaeast", "eastus", "japaneast", "westus"]}}}, {"policyDefinitionId": "/providers/Microsoft.Management/managementgroups/cli-test-mgmt-group000002/providers/Microsoft.Authorization/policyDefinitions/azure-cli-test-data-policy000005"}]
```

Comment:
Is it used for test? It's weird to modify it instead of adding new json.

Reply:
Do we need some tests for your new argument?

Reply:
I have discussed this with yugang, and we think it is not necessary to add test for this argument. And I have test it locally and this argument is already added for create command.

Reply:
> Is it used for test? It's weird to modify it instead of adding new json.

Every time you run azdev test resource, this file will change... It is not for this argument.